src/components/accounts/CreateAccount.js

```diff
@@ -48,12 +50,25 @@ class CreateAccount extends Component {
         })
     }
 
-    handleRecaptcha = value => {
-        console.log(value)
+    verifyRecaptchaThree = (token) => {
```

Comment:
@Patrick1904  recaptcha verification has to happen on backend at the same time as account creation request is issued. Otherwise we'll need to maintain server-side session and authorize creation request based on previous separate request to verify captcha (this is suboptimal).

So I think we should use such flow:
1) Send account creation request with V3 token.
2) If server returns success – we are done account creation. 
3) Otherwise (if error is https://httpstatuses.com/402) – present V3 recaptcha to user and resubmit request.
4) Repeat 3) until server returns success for account creation.